elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -516,14 +524,34 @@ def categorize_bugs_by_type(
 
     not_found = set(tracker_bugs) - found
     if not_found:
-        still_not_found = not_found
+        # Filter out JIRA bugs with security levels not in the allowlist
+        filtered_not_found = set()
+        security_filtered_bugs = set()
+        
+        for bug in not_found:
```

Comment:
This may work, but I would have imagined this to be implemented in the `bzutil.JIRABugTracker._query()` method. This is the global entrypoint for all queries. Did you look into formulating this exclusion in terms of jql?

Reply:
No I didn't, but I agree this will filter the bugs at the start so, we would need to crunch through less bugs. I will look into this.